hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlan.java

```diff
@@ -335,10 +335,10 @@ private void initProcSuppliers(long jobId, long executionId) {
 
     private void initDag() {
         final Map<Integer, VertexDef> vMap = vertices.stream().collect(toMap(VertexDef::vertexId, v -> v));
-        vertices.forEach(v -> {
+        for (VertexDef v : vertices) {
```

Comment:
why change the forEach here? technically it provides more efficient iteration then `for ()` since it doesn't create an iterator

Reply:
OTOH it doesn't create a lambda, so in terms of allocations it's the same. I like the old school way, it's easier to debug, stack trace is shorter.